extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTableWriter.java

```diff
@@ -920,20 +919,22 @@ public void close() {
         }
     }
 
-    static class ShortTransfer implements TransferObject<IntBuffer> {
+    /**
+     * Used as a base class of transfer objects for types where we don't enforce any page size limits
+     */
+    abstract static class BaseTransfer<T extends ChunkBase<Values>> implements TransferObject<IntBuffer> {
```

Comment:
I don't think we need TransferObject and BaseTransfer.  You can make TransferObject an abstract class instead of an interface and move this common functionality to it.